src/pathfinding_service/api.py

```diff
@@ -459,6 +459,32 @@ def get(self) -> Tuple[dict, int]:
         return info, 200
 
 
+class UserResource(PathfinderResource):
+    def get(self, user_address: str) -> Tuple[Dict[str, str], int]:
+        address = self._validate_address_argument(user_address)
+        user_manager = self.pathfinding_service.matrix_listener.user_manager
+        user_ids = user_manager.get_userids_for_address(address)
+        for user_id in user_ids:
+            if user_manager.get_userid_presence(user_id) in [
+                UserPresence.ONLINE,
+                UserPresence.UNAVAILABLE,
```

Comment:
What does UNAVAILABLE mean, again? It doesn't sound like it should be considered the same way as ONLINE.

Reply:
It does. Unavailable is something like `invisible` but it is considered to be online. A bit weird naming though.